config/rbac/role.yaml

```diff
@@ -257,6 +264,15 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ocs.openshift.io
+  resources:
+  - storagerequests
+  verbs:
+  - delete
+  - get
+  - list
+  - watch
```

Comment:
Shouldn't we be removing any RBAC related to storage request instead of adding?

Reply:
Replied here 
https://github.com/red-hat-storage/ocs-operator/pull/3070#discussion_r2063294802